userguide/directoryservice.rst

```diff
@@ -40,23 +40,22 @@ support within %brand%.  It is strongly recommended to update the
 system to the latest %brand% |release| before attempting Active
 Directory integration.
 
-**Before configuring the Active Directory service**, ensure name
-resolution is properly configured by :command:`ping` ing the domain
-name of the Active Directory domain controller from :ref:`Shell` on
-the %brand% system. If the :command:`ping` fails, check the DNS
-server and default gateway settings in
-:menuselection:`Network --> Global Configuration`
-on the %brand% system.
+Before configuring the Active Directory service, ensure name resolution
```

Comment:
This sentence is really, really long and has lots of clauses.  Please shorten or break up.